ThePhasesOfCraftship/2_best_practice_first/strategicDesignPart1/exercises/1_RefactoringTo4Tiers/begin/src/services/createAssingmentService.ts

```diff
@@ -0,0 +1,90 @@
+import { Database } from "../persistence";
+import { parseForResponse } from "../utils";
+import { ErrorExceptionType } from "../constants";
+import { AssignmentRequestDto } from "../dto/AssignmentRequestDto";
+
+export interface IAssignmentService {
+  createAssignment: (dto: AssignmentRequestDto) => Promise<any>;
```

Comment:
Same here, I'd suggest using strictly typed return types.

Reply:
Yes, I agree. Do you think I should do something like `=> Promise<Assignment>` where the `Assignment` type is generated by Prisma?

Reply:
Exactly, you could just `import {Assignment} from '@prisma/client'`